src/class-parsely.php

```diff
@@ -918,7 +951,30 @@ public function construct_parsely_metadata( array $parsely_options, $post ) {
 			$tags = array_map( array( $this, 'get_clean_parsely_page_value' ), $tags );
 			$tags = array_values( array_unique( $tags ) );
 
-			$parsely_page['@type']            = 'NewsArticle';
+			/**
+			 * Filters the JSON-LD @type.
+			 *
+			 * @since 2.5.0
+			 *
+			 * @param array   $jsonld_type  JSON-LD @type value, default is NewsArticle.
+			 * @param integer $id           Post ID.
+			 * @param string  $post_type    Post type in WordPress.
+			 */
+			$type          = apply_filters( 'wp_parsely_post_type', 'NewsArticle', $post->ID, $post->post_type );
```

Comment:
```suggestion
			$type          = (string) apply_filters( 'wp_parsely_post_type', 'NewsArticle', $post->ID, $post->post_type );
```

Let's ensure that we're dealing with the type we think we're dealing with.